airflow/www/views.py

```diff
@@ -2392,6 +2392,7 @@ def extra_links(self):
         task_id = request.args.get('task_id')
         execution_date = request.args.get('execution_date')
         link_name = request.args.get('link_name')
+        link_names = request.args.getlist('link_name')
```

Comment:
There is no point in keeping `link_name = ...` here any more. I do like the `link_names` variable name for the list!